src/tasks/autounexile.py

```diff
@@ -0,0 +1,24 @@
+from discord.ext import tasks
+from discord.ext.commands import Bot
+from database import exiles_database
+from services.exile_service import unexile_user
+from settings import get_settings
+from .helper import create_autounexile_embed
+
+settings = get_settings()
+
+
+@tasks.loop(seconds=5.0)
```

Comment:
if performance gets shot a little bit we can extend this loop to be every 30 secs - 1 min